dev-util/hip/hip-5.0.2.ebuild

```diff
@@ -79,7 +79,7 @@ src_prepare() {
 	hprefixify $(grep -rl --exclude-dir=build/ "/usr" "${S}")
 	hprefixify $(grep -rl --exclude-dir=build/ "/usr" "${HIP_S}")
 
-	cp "$(prefixify_ro "${FILESDIR}"/hipvars.pm)" bin/ || die "failed to replace hipvars.pm"
+	cp "$(prefixify_ro "${FILESDIR}"/hipvars-5.0.2.pm)" bin/hipvars.pm || die "failed to replace hipvars.pm"
```

Comment:
use `dobin` and leave out `|| die`

Reply:
I think `dobin` is an install function while this is `src_prepare`?